### DISCUSSION TITLE
ML Developer Advocate Opportunity at Deepgram

### DISCUSSION THREAD

**Discussion by WBonnheim** [deepgram/community#53] [General help]
We are thrilled to share that we're adding an ML Developer Advocate to our DevRel team. We're looking for an ML Developer Advocate that is passionate about machine learning and voice. They'll join a team of engineers and storytellers representing Deepgram to developer and data science communities and inspiring them to build amazing products. The full job description can be found here - https://jobs.lever.co/deepgram/f20abb11-d158-4a90-9f6d-f54a30fa6e24. Don't hesitate to reach out with any questions or with referrals. Let's build Deepgram together!

**Comment by realray808** [comment]
您的邮件已经收到 谢谢
